mm-redmine.psm1

```diff
@@ -179,7 +179,8 @@ function Update-RedmineIssue {
         [int[]]  $WatcherId,
         [string] $Notes,
         [switch] $PrivateNotes,
-        [array]  $Uploads
+        [array]  $Uploads,
+        [array]  $CustomFields
```

Comment:
It can't be done half baked like this. There is also `New-RedmineIssue` function